packages/hash/frontend/src/graphql/queries/account.queries.ts

```diff
@@ -41,6 +41,18 @@ export const getAccountPages = gql`
   }
 `;
 
+export const getAccountPagesTree = gql`
+  query getAccountPagesTree($accountId: ID!) {
+    accountPages(accountId: $accountId, structure: Tree) {
+      entityId
+      properties {
+        title
+      }
+      parentPageId
```

Comment:
This is now `parentPageEntityId` on main

Reply:
Thanks!